Android/ZensoftHRCRM/app/src/main/res/layout/fragment_date_picker.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
+
+
+  <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center_horizontal"
+      android:orientation="vertical">
+    <DatePicker
+        android:id="@+id/crint_date_picker"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+    <TimePicker
+        android:id="@+id/crint_time_picker"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+  </LinearLayout>
+</ScrollView>
```

Comment:
Why do we need scrollView here?

Reply:
Date picker and Time picker in one fragment take a lot of space in new android versions